### PR TITLE
{tools}[gompi/2019a,iimpi/2019a] mpifileutils v0.9.1

### PR DESCRIPTION
**Pull request by JackPerdue**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/c598a879fd0fc6d7690303ea53a5e0a3 for a full test report.

File: easybuild/easyconfigs/m/mpifileutils/mpifileutils-0.9.1-intel-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+name = 'mpifileutils'
+version = '0.9.1'
+
+homepage = 'https://hpc.github.io/%(name)s/'
+
+description = """
+ MPI-Based File Utilities For Distributed Systems
+"""
+
+toolchain = {'name': 'intel', 'version': '2019a'}
```
Comment (boegel) [inline easybuild/easyconfigs/m/mpifileutils/mpifileutils-0.9.1-intel-2019a.eb]: @JackPerdue Can you clarify why you're not using `iimpi` here?

File: easybuild/easyconfigs/m/mpifileutils/mpifileutils-0.9.1-intel-2019a.eb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+source_urls = ['https://github.com/hpc/%(name)s/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['15a22450f86b15e7dc4730950b880fda3ef6f59ac82af0b268674d272aa61c69']
+
+osdependencies = ['libattr-devel']
```
Comment (boegel) [inline easybuild/easyconfigs/m/mpifileutils/mpifileutils-0.9.1-intel-2019a.eb]: Is providing `libattr` through EasyBuild not an option?

File: easybuild/easyconfigs/m/mpifileutils/mpifileutils-0.9.1-gompi-2019a.eb [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+checksums = ['15a22450f86b15e7dc4730950b880fda3ef6f59ac82af0b268674d272aa61c69']
+
+builddependencies = [
+    ('CMake', '3.13.3', '', ('GCCcore', '8.2.0')),
+]
+dependencies = [
+    ('attr', '2.4.47', '', ('GCCcore', '8.2.0')),
+    ('bzip2', '1.0.6', '', ('GCCcore', '8.2.0')),
+    ('dtcmp', '1.1.0'),
+    ('libarchive', '3.4.0', '', ('GCCcore', '8.2.0')),
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/m/mpifileutils/mpifileutils-0.9.1-gompi-2019a.eb]: specifying the subtoolchain isn't necessary, is it?

Comment (boegel) [reply easybuild/easyconfigs/m/mpifileutils/mpifileutils-0.9.1-gompi-2019a.eb]: Indeed, the hardcoded toolchains should be removed...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 10 out of 10 (10 easyconfigs in this PR)
node3103.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/e24c9192a16d3f100bcb2bc39370bf14 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 10 out of 10 (10 easyconfigs in this PR)
node2470.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/1174fc16b19f7b8bc604d2c85fe8d17f for a full test report.

**Comment by boegel** [comment]
Going in, thanks @JackPerdue!

**Pull request merged**